legal-api/src/legal_api/services/filings/validations/common_validations.py

```diff
@@ -165,6 +168,12 @@ def validate_court_order(court_order_path, court_order):
     """Validate the courtOrder data of the filing."""
     msg = []
 
+    required_permission = ListActionsPermissionsAllowed.COURT_ORDER_POA.value
+    message = f'Permission Denied - You do not have permissions to add or change a court order in this filing.'
+    error = PermissionService.check_user_permission(required_permission, message)
```

Comment:
I don't know the cost of creating 2  new variables (abeit static text), but wouldn't this be cleaner and possibly faster?

```
error = PermissionService.check_user_permission(
   ListActionsPermissionsAllowed.COURT_ORDER_POA.value,
   f'Permission Denied - You do not have permissions to add or change a court order in this filing.'
)
```